File: public/example_code/python/example_02/person.py
#person.py
class Person(object):

    def __init__(self,h):
        self.height = h

    def printHeight(self):
        print "Height: ",self.height


def main():
    guy = Person(5)
    guy.printHeight()

if __name__ == "__main__":
    main()

File: public/example_code/java/example_08/Person.java
//Person.java
class Person {

    private int height;

    public Person(int h) {
      height = h;
    }

    public void printHeight() {
      System.out.println("Height: "+ height);
    }

    public static void main(String[] args) {
        Person guy = new Person(5);
        guy.printHeight();
    }
}
